$OpenBSD: patch-config_mk,v 1.12 2022/02/19 21:30:49 bcallah Exp $

zap redundant CFLAGS

fix "filelist.c:253:45: error: use of undeclared identifier 'alphasort'"

Index: config.mk
--- config.mk.orig
+++ config.mk
@@ -36,10 +36,6 @@ scalable_icon_dir = ${icon_dir}/scalable/apps
 
 # default CFLAGS
 CFLAGS ?= -g -O2
-CFLAGS += -Wall -Wextra -pedantic
-
-# Settings for glibc >= 2.19 - may need to be adjusted for other systems
-CFLAGS += -std=c11 -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700
 
 ifeq (${curl},1)
 	CFLAGS += -DHAVE_LIBCURL
